homeassistant/components/deconz/config_flow.py

```diff
@@ -79,7 +74,11 @@ def __init__(self):
         if user_input is not None:
             for bridge in self.bridges:
                 if bridge[CONF_HOST] == user_input[CONF_HOST]:
-                    self.deconz_config = bridge
+                    self.bridgeid = bridge[CONF_BRIDGEID]
```

Comment:
Why break up the `bridge` object here in 2 parts, can't we just store the whole `bridge` object ?

Reply:
That's what's gonna get stored in config entry data. Trying to keep it to what's expected